doc/jedi-vim.txt

```diff
@@ -274,12 +283,9 @@ This function finds the first definition of the function/class under the
 cursor. It produces an error if the definition is not in a Python file.
 
 ------------------------------------------------------------------------------
-5.4. `g:jedi#goto_definitions_command`  *g:jedi#goto_definitions_command*
-Function: `jedi#goto_definitions()`
-Default: -                              Go to original definition
-
-NOTE: Deprecated. Use |g:jedi#goto_command| / |jedi#goto()| instead, which
-currently uses this internally.
+5.4. `g:jedi#goto_stubs_command`  *g:jedi#goto_stubs_command*
+Function: `jedi#goto_stubs_command()`
+Default: <leader>s                              Go to stubs definition
```

Comment:
Not sure about the correct wording with regard to stubs.
Also this could have a paragraph describing it, too.